http/swagger.yml

```diff
@@ -4984,6 +5100,30 @@ components:
           description: A description of the event that occurred.
           type: string
           example: Halt and catch fire
+    OperationLog:
+      type: object
+      readOnly: true
+      properties:
+        description:
+          type: string
+          description: A description of the event that occurred.
+          example: Bucket Created
+        time:
+          type: string
+          description: Time event occurred, RFC3339Nano.
+          format: date-time
+        links:
+          type: object
+          properties:
+            user:
+              $ref: "#/components/schemas/Link"
+    OperationLogs:
+      type: object
+      properties:
+        logs:
+          $ref: "#/components/schemas/OperationLog"
```

Comment:
@desa should this be an array of operation logs?

Reply:
Yup. Array seems right to me.

Reply:
@zhulongcheng would you change this to an array rather than an object?